proxy/src/main/java/com/wavefront/agent/ProxyConfig.java

```diff
@@ -1739,12 +1739,6 @@ public void verifyAndInit() {
       token = ObjectUtils.firstNonNull(config.getRawProperty("token", token), "undefined").trim();
       server = config.getString("server", server);
       hostname = config.getString("hostname", hostname);
-      if (hostname != null) {
```

Comment:
we still need this.... the thing we don't need is the else because is done on line 549

Reply:
If we set the value of hostname on L#549, `hostname != null` will always be true, and the log statement will be printed even if hostname is not specified in config.ini. @laullon , should I revert to keeping the else statement?